docs/content/docs/dev/table/sql/queries/hints.md

```diff
@@ -84,4 +84,205 @@ insert into kafka_table1 /*+ OPTIONS('sink.partitioner'='round-robin') */ select
 
 ```
 
+## Query Hints
+
+`Query Hints` specify that the indicated hints are used in the scope of a query. They affect all operators in the statement.
```

Comment:
It's necessary to clarify the effective scope of a query hint, especially the mechanism of hints propagation, this is important for users.

Reply:
> It's necessary to clarify the effective scope of a query hint, especially the mechanism of hints propagation, this is important for users.

Done!